torch/_inductor/kernel/mm.py

```diff
@@ -199,3 +201,18 @@ def tuned_addmm(inp, mat1, mat2, *, alpha=1, beta=1, layout=None):
     return autotune_select_algorithm(
         "addmm", choices, [inp_expanded, mat1, mat2], layout
     )
+
+
+def tuned_mixed_mm(mat1, mat2, mat2_dtype):
+    m, n, k, layout, mat1, mat2 = mm_args(mat1, mat2, layout=None)
+    # options to tune from
+    choices = []
```

Comment:
Should we have a choice to fallback to aten (with two kernels)?

Reply:
how would i do that? atm its gated by config only